docs/_posts/2015-07-03-react-v0.14-beta-1.md

```diff
@@ -0,0 +1,73 @@
+
+The `react` package contains `React.createElement`, `React.createClass` and `React.Component`, `React.PropTypes`, `React.Children`, and the other helpers related to elements and component classes. We think of these as the [_isomorphic_](http://nerds.airbnb.com/isomorphic-javascript-future-web-apps/) or [_universal_](https://medium.com/@mjackson/universal-javascript-4761051b7ae9) helpers that you need to build components.
+
+The `react-dom` package contains `ReactDOM.render`, `ReactDOM.unmountComponentAtNode`, and `ReactDOM.findDOMNode`, and in `react-dom/server` we have server-side rendering support with `ReactDOMServer.renderToString` and `ReactDOMServer.renderToStaticMarkup`.
+
+```js
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var MyComponent = React.createClass({
+  render: function() {
+    return <div>Hello World</div>;
+  }
+});
+
+ReactDOM.render(<MyComponent />, node);
+```
+
+We anticipate that most components will need to depend only on the `react` package, which is lightweight and doesn't include any of the actual rendering logic. To start, we expect people to render DOM-based components with our `react-dom` package, but there's nothing stopping someone from diving deep on performance and writing a `awesome-faster-react-dom` package which can render *the exact same DOM-based components*. By decoupling the component definitions from the rendering, this becomes possible.
+
+Better yet, this paves the way to writing components that can be shared between the web version of React and React Native. This isn't yet easily possible, but we intend to make this easy in a future version so you can share React code between your website and native apps.
+
+The addons have moved to separate packages as well: react-addons-clone-with-props, react-addons-create-fragment, react-addons-css-transition-group, react-addons-linked-state-mixin, react-addons-pure-render-mixin, react-addons-shallow-compare, react-addons-transition-group, and react-addons-updates, plus `ReactDOM.unstable_batchedUpdates` in react-dom.
+
+For now, please use the same version of `react` and `react-dom` in your apps to avoid versioning problems -- but we plan to remove this requirement later. (This release includes the old methods in the `react` package with a deprecation warning, but they'll be removed completely in the next release.)
+
+## DOM node refs
+
+The other big change we're making in this release is exposing refs to DOM components as the DOM node itself. That means: we looked at what you can do with a `ref` to a DOM component and realized that the only useful thing you can do with it is call `this.refs.giraffe.getDOMNode()` to get the underlying DOM node. In this release, `this.refs.giraffe` _is_ the actual DOM node.
+
+```js
+var Zoo = React.createClass({
+  render: function() {
+    return (
+      <div>
+        Giraffe's name: <input ref="giraffe" />
```

Comment:
@mathieumg I suspect because github doesn't understand jsx. It hardly matters. People won't be reading the raw github markdown file anyways.